pkg/watch/term_unix_other.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build aix || linux || solaris || zos
+// +build aix linux solaris zos
+
+package watch
+
+import "golang.org/x/sys/unix"
+
+const ioctlReadTermios = unix.TCGETS
+const ioctlWriteTermios = unix.TCSETS
```

Comment:
What are these constants for exactly? 
What's the significance of this file and the file `term_unix_bsd.go`?

Reply:
They are the constants for Terminal I/Os on Unix. Unfortunately, constants values are different on BSD and on other Unixes, thus the 2 different files with different constants.